extensions/wikia/LookupContribs/LookupContribs.php

```diff
@@ -1,8 +1,71 @@
 <?php
-define ( 'LOOKUPCONTRIBS_HELP', 'This special page can be used to display user\'s contributions on multiple wikias. For displaying the list, give username below.' ) ;
-define ( 'LOOKUPCONTRIBS_PATH', '/' ) ;
-/* cache? no cache? */
-define ( 'LOOKUPCONTRIBS_NO_CACHE', 'false' ) ;
 
-require_once ( $IP . LOOKUPCONTRIBS_PATH . "extensions/wikia/LookupContribs/SpecialLookupContribs.php" ) ;
-require_once ( $IP . LOOKUPCONTRIBS_PATH . "extensions/wikia/LookupContribs/SpecialLookupContribs_hooks.php" ) ;
```

Comment:
Not sure why we need to require these and separately.  Pulled the contents of these two files into this single file and deleted the other two.